packages/suite/src/components/suite/modals/TransactionDetail/components/AmountDetails/index.tsx

```diff
@@ -89,6 +89,27 @@ const AmountDetails = ({ tx, txDetails, isTestnet }: Props) => {
             ? formatNetworkAmount(totalOutput.toFixed(), tx.symbol)
             : undefined;
 
+    /* 
+    Get the Amount value to show if the user sent transaction to himself.
+
+    tx.amount value returns how much the user's wallet balance changed. 
+    But if the user sent transaction to himself, tx.amount returns only the cost of tx.fee. That's valid behavior, but confusing. 
+
+    I couldn't just use amountSentToSelf=totalInput, because totalInput is undefined for Ethereum (is that a bug or valid behavior?).
+    For that reason I added together the value of txFee and totalOutput instead, because that works properly for all currently supported coins. 
+    */
+    const txSentToSelf = tx.type === 'self';
+    let amountSentToSelf = '';
+
+    if (txSentToSelf && network && totalOutput && !totalOutput.isNaN()) {
+        // convert float value of fees back to the absolute value in basic network units in order to prevent possible errors coming from summing up float numbers
+        const feesAbsolute = Number(tx.fee) * 10 ** network.decimals;
+        // get the formatted value of amountSentToSelf
+        amountSentToSelf = feesAbsolute
```

Comment:
I don't think a ternary is necessary here as the variable already has an empty string assigned to it when initialized. I would transform this into an if and set its value if the condition is true.